bin/alluxio-start.sh

```diff
@@ -35,9 +35,8 @@ Where ACTION is one of:
   restart_workers    \tRestart any failed workers on worker nodes.
 
 MOPT (Mount Option) is one of:
-  Mount    \tMount the configured RamFS. Notice: this will format the existing RamFS.
-  SudoMount\tMount the configured RamFS using sudo.
-           \tNotice: this will format the existing RamFS.
+  Mount    \tMount the configured RamFS if it is not already mounted.
+  SudoMount\tMount the configured RamFS using sudo if it is not already mounted.
```

Comment:
Is the description still accurate?

Reply:
I believe so, we recently changed the behavior to not re-mount if a ramfs is already there: https://alluxio.atlassian.net/browse/ALLUXIO-3257

Reply:
That change is why the ability to manually un-mount is much more important now